model_hub/examples/mmdetection/fasterrcnn.yaml

```diff
@@ -15,35 +15,35 @@ data:
 #  end_after_batch: 300
 hyperparameters:
   global_batch_size: 16
-  validation_batch_size_per_gpu: 2
-  config_file: /mmdetection/configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py
+  config_file: /mmdetection/configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_1x_coco.py
   merge_config: null # You can specify a config you want to merge into the config_file above.
   use_pretrained: false # Whether to load pretrained weights for config if available.
-  override_config_fields:
+  override_mmdet_config:
     ##### Learn more about mmdet configs: https://mmdetection.readthedocs.io/en/latest/tutorials/config.html #####
-    data.train.ann_file: /tmp/instances_train2017.json
-    data.val.ann_file: /tmp/instances_val2017.json
     ##### You can specify gradient clipping with below #####
-    #optimizer_config._delete_: true
-    #optimizer_config.grad_clip.max_norm: 35
-    #optimizer_config.grad_clip.norm_type: 2
+    optimizer_config._delete_: true
+    optimizer_config.grad_clip.max_norm: 35
+    optimizer_config.grad_clip.norm_type: 2
+    lr_config.warmup_ratio: 0.3333333
+    ##### You can specify mixed precision with below #####
+    #fp16.loss_scale: 512. # can be float or dict of named args for torch.cuda.amp.GradScaler
 
 checkpoint_storage:
   save_trial_latest: 5
 min_validation_period:
-  batches: 7320
+  batches: 7330
 searcher:
   name: single
   metric: bbox_mAP
   max_length:
-    batches: 87850
+    batches: 87960
   smaller_is_better: false
-max_restarts: 0
+max_restarts: 5
 environment:
   image:
-    gpu: determinedai/model-hub-mmdetection
+    gpu: determinedai/model-hub-mmdetection:0.17.0.dev0
```

Comment:
Should use commit id.

Reply:
we push tip of master to the dev tag every commit.  on release, the dev suffix is removed.  the assumption is that users will checkout a release tag or just download the example from our doc.

Reply:
> on release, the dev suffix is removed.

This won't be done automatically. Is the plan to remove the tag?

Reply:
This is done automatically as part of bumpenvs run during release process. It's one reason we need the dev version here.